src/main/java/seedu/duke/commands/HomeCommand.java

```diff
@@ -0,0 +1,31 @@
+package seedu.duke.commands;
+
+import seedu.duke.Parser;
+import seedu.duke.project.Project;
+
+import java.util.ArrayList;
+
+public class HomeCommand extends Command {
+
+    private int projectIndex;
+
+    public HomeCommand(int projectIndex) {
+        this.projectIndex = projectIndex;
+    }
+
+    @Override
+    public String executeCommand(ArrayList<Project> projects) {
+        if (projectIndex == -1) {
+            return "Already in Home View!";
```

Comment:
Should we extract this to Ui class or Exception class??

Reply:
Oh thanks for the review, just committed the changes in Home Command PR!